content/en/docs/concepts/storage/volumes.md

```diff
@@ -121,18 +121,18 @@ If the EBS volume is partitioned, you can supply the optional field `partition:
 
 #### AWS EBS CSI migration
 
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
 
 The `CSIMigration` feature for `awsElasticBlockStore`, when enabled, redirects
 all plugin operations from the existing in-tree plugin to the `ebs.csi.aws.com` Container
 Storage Interface (CSI) driver. In order to use this feature, the [AWS EBS CSI
 driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
 must be installed on the cluster and the `CSIMigration` and `CSIMigrationAWS`
-beta features must be enabled.
+features must be enabled.
```

Comment:
```suggestion
must be installed on the cluster and the `CSIMigration` feature must be enabled.
```
AIUI, you can't disable `CSIMigrationAWS` any more.

Reply:
you are right, CSIMigration is also locked to true in 1.25 https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L884, lemme just remove the 'and the x feature must be enabled'